### DISCUSSION TITLE
浮动气泡在一个iframe上，无法点击及移动

### DISCUSSION THREAD

**Discussion by xiahller** [ant-design/ant-design-mobile#5621] [Q&A]
一个页面中嵌入了一个iframe页面，加了一个用来收藏本页面的浮动气泡，当把浮动气泡拖入iframe的区域内，就无法点击浮动气泡，只能点击拖拽iframe内的内容了，用阻止冒泡也没有办法解决。

**Comment by awmleer** [comment]
有点奇怪，可以提供一个复现 demo 么？
